.github/workflows/deploy-testnet.yml

```diff
@@ -0,0 +1,43 @@
+name: Testnet CD
+
+on:
+  push:
+    tags:
+      - '**-beta'
```

Comment:
Question: should be this something like `**-beta*`? It should work with any beta tag, so tags like `1.0.0-beta.x` should be taken into account.
I'm not sure if with the filter `**-beta` we take into account `x.y.z-beta` only.

Reply:
Okay, I will update that.

Reply:
Updated